source/DataHubClient.ts

```diff
@@ -61,45 +106,43 @@ export default class DataHubClient {
 					  domain {
 						id
 					  }
-					}	
+					}
 				  }
 				`,
 			}),
-		})
-			.then((r) => r.json());
+		}).then((r) => r.json());
 	};
 
-	private _searchFullTextWithAuth = async (type: "DATASET", searchString: string, token: string) => {
-		return fetch(this._graphQlEndpoint, {
-			method: "POST",
-			headers: {
-				authorization: `Bearer ${token}`,
-				"Content-Type": "application/json",
-				Accept: "application/json",
-			},
-			body: JSON.stringify({
-				query: `
-				{
-					search(input: { type: ${type}, query: "${searchString}", start: 0, count: 10 }) {
-					start
-					count
-					total
-					searchResults {
-						entity {
-						urn
-						type
-						...on Dataset {
-							name
-						}
-						}
-					}
-					}
-				}
-			`,
-			}),
-		})
-			.then((r) => r.json());
-	};
+	// private _searchFullTextWithAuth = async (type: "DATASET", searchString: string, token: string) => {
+	// 	fetch(this._graphQlEndpoint, {
+	// 		method: "POST",
+	// 		headers: {
+	// 			authorization: `Bearer ${token}`,
+	// 			"Content-Type": "application/json",
+	// 			Accept: "application/json",
+	// 		},
+	// 		body: JSON.stringify({
+	// 			query: `
+	// 			{
+	// 				search(input: { type: ${type}, query: "${searchString}", start: 0, count: 10 }) {
+	// 				start
+	// 				count
+	// 				total
+	// 				searchResults {
+	// 					entity {
+	// 					urn
+	// 					type
+	// 					...on Dataset {
+	// 						name
+	// 					}
+	// 					}
+	// 				}
+	// 				}
+	// 			}
+	// 		`,
+	// 		}),
+	// 	}).then((r) => r.json());
+	// };
```

Comment:
I think we can uncomment that and just not use this method for now. It can be unclear for some future developers why this was commented out

Reply:
done, thanks :)